### PR TITLE
feat(TIC): placeholder

### PR DESCRIPTION
**Pull request by barbnewrelic**
For DOC7761

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/nav/logs.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@ pages:
         path: /docs/logs/get-started/get-started-log-management
       - title: Log management and security
         path: /docs/logs/get-started/new-relics-log-management-security-privacy
+  - title: Logs everywhere
+    pages:
+      - title: Troubleshooting with telemetry in contextt
```
Comment (paperclypse) [inline src/nav/logs.yml:12]:
```suggestion
      - title: Troubleshooting with telemetry in context
```

File: src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+    ![Error logs](./images/demotron5-logs030722.png "Error logs")
+
+    </Collapser>
+
+    <Collapser
+    id="log-sampling"
+    title="What additional test data will help troubleshoot?"
+    >
+
+    The troubleshooting team has been running tests to isolate the cause, but not all logs have been collected for the test transactions. The `AlwaysOn` default log collection has been useful to validate that the services have an issue and to examine it initially. But now they need to turn up the sampling to continue troubleshooting.
```
Comment (kaylareopelle) [inline src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx:89]: What does `AlwaysOn` refer to? 

If you'd like to add the process to turn up sampling: 

To adjust their sampling, they'll update the configuration `application_logging.forwarding.max_samples_stored` to a higher number. It defaults to 10,000 logs stored per minute.

File: src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+    
+    They also update their runbooks to point to the log patterns page filtered to this app, and [add the runbook to their alert condition](/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/understand-technical-concepts/provide-runbook-instructions-alert-activity/). Next time they get an alert similar to this situation, they can use these lessons learned for faster troubleshooting.
+
+    </Collapser>
+
+</CollapserGroup>
+
+## Supported agent versions [#agents]
+
+Applications don't exist in isolation, and the resources you use to monitor, troubleshoot, and improve their performance are both time-critical and complex. You need a simple yet holistic solution to quickly see and act on all your telemetry in context.
```
Comment (kaylareopelle) [inline src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx:108]:
```suggestion
Applications don't exist in isolation. The resources you use to monitor, troubleshoot, and improve their performance are both time-critical and complex. You need a simple yet holistic solution to quickly see and act on all your telemetry in context.
```

File: src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+You can also add more attributes to make your troubleshooting focus even more precise. For example, you may want to add more attributes to get other key telemetry data specific to your ecosystem. 
+
+**REVIEWER: Can you give a few examples of the types of attributes I might want to add? Is this referring to [infrastructure log's optional configuration documentation](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent#optional-config), or are there other things they might want to add?**
+
+## Avoid ingest limits and extra billing [#ingest]
+
+Using telemetry in context functionality will increase your data ingest, and this may have an impact on your ingest limits and billing. For example, when your engineering team is troubleshooting a problem with your app, it may be helpful to temporarily set the limits to maximum data collection. (By default, the maximum number of logs sampled is 10,000. **REVIEWER: Is 10,000 max correct?**) But if you leave this running for several days, this could result in a lot of unnecessary data and wasted money.
```
Comment (kaylareopelle) [inline src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx:144]: Yes, 10,000 is correct! (Though technically, for the Ruby Agent the closest divisor is 9,996)

File: src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+
+If you're troubleshooting an error with your app, you can easily see related network activity, Kubernetes pod health, cloud provider information, database storage, load balancer resources, logs, transaction data, spans and traces, and more, all in the context of the APM error.
+
+With telemetry in context, now your logs are:
+
+* **Time-sensitive.** See the counts and percentages of logs associated with your apps, hosts, and other related entities for a specific timeframe. Quickly gain context at that point in time.
+* **Precise.** Cut through the noise of thousands of logs when troubleshooting time-critical issues, so you automatically see only the most relevant logs. Navigate within multiple types of telemetry data and have them correlate back to the original issue.
+* **Security minded.** Understand context for security information and event management (SIEM) data across your infrastructure, network performance management, cloud platforms, and more.
```
Comment (kaylareopelle) [inline src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx:213]: I'm not sure if this is quite relevant?

File: src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx [hunk 6]
```diff
@@ -0,0 +1,219 @@
...
+
+![Telemetry in context](./images/demotron6-logs030722.png "Telemetry in context")
+
+<figcaption>
+  Drill down into your logs, errors, traces, and spans, all from the APM **Summary** page in New Relic One.
+</figcaption>
+
+More importantly, when a problem arises, you immediately have an overall view of the health of **all** your systems, regardless of the type of data in those individual systems and components. This also makes it easier to link your log telemetry data across accounts.
+
+Best of all, there's no additional configuration required.
```
[the remark this answers is not in the document]
Comment (barbnewrelic) [reply src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx:20]: Thanks for clarifying!

File: src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+
+  **TO DO: Update screenshots in this example throughout the journey.**
+
+  When the engineer looks at the app's **Summary** page, they see many more error logs leading up to the increase in response time. They want to see if the logs can expose the root cause of the problem.
+
+  ![APM summary errors](./images/demotron1-logs030722.png "APM summary errors")
+
+  **How can they quickly decide what's related to the problem?**
+  
+  They have not set up log collection. But the Ruby agent for their app has been recently updated to include telemetry in context, so the app now automatically receives metrics about its logs. The metrics are faceting the count by a severity field commonly included in their Rails framework.
```
[the remark this answers is not in the document]
Comment (barbnewrelic) [reply src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx:44]: Thank you!

File: src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx (hunk 2 above)
Comment (barbnewrelic) [reply src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx:89]: When agent teams were talking about the default being always on, and I think I interpreted that as a config value called AlwaysOn. I'll double-check my notes, and in the meantime I will add your recommendation. Thanks for making this accurate!

File: src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx (hunk 5 above)
Comment (barbnewrelic) [reply src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx:213]: Agreed. I was throwing a lot of separate pieces of info at this draft. :-)

**Pull request closed without merging**